system/modules/core/library/Contao/File.php

```diff
@@ -280,16 +280,6 @@ public function __get($strKey)
 								\Image::getPixelValue($svgElement->getAttribute('height'))
 							);
 						}
-						elseif ($svgElement->getAttribute('viewBox'))
```

Comment:
I explained it in https://github.com/contao/core/pull/7882#issue-87808347 `imageSize` must only respect fixed dimensions, not percentage or viewBox based values. I added `imageViewSize` which takes the `viewBox` into account if `imageSize` is not defined.